### PR TITLE
Edited-Create-an-Account

### PR DESCRIPTION
**Pull request by 0101Diana**
Created a brief demo video and inserted (most people are visual learners, today, so though that placing it on top was the better option, followed up with the a writing section.

Updated/Changed pictures according to specifications and with what Fabienne had.

Rewrote/edited document.

### CODE REVIEW

File: content/help/manual/create_account/index.md [hunk 1]
```diff
@@ -3,106 +3,77 @@ title: Creating An Account
...
-remember. We know this is not an easy task, so the app will assist you by
-suggesting a strong password for you to use. This is a secure passphrase that
-is randomly generated from a list of the most common words in the English
-language (or whichever language you have set in your system preferences, as
-long as it is supported) using a variation of the [diceware
-method](https://ssd.eff.org/en/module/animated-overview-how-make-super-secure-password-using-dice).
-This passphrase may not look particularly secure, but we assure you it is
-likely much better than anything you would come up by yourself (no offence) and
-it certainly is a lot easier to remember than a string of random characters!
+This is the last password you **HAVE TO REMEMBER** and will be the most important password that you **NEED TO REMEMBER**.
```
Comment (MaKleSoft) [inline content/help/manual/create_account/index.md]: Is there a reason why you got rid of all this text? The manual is supposed to convey more information than just step-by-step instructions, and this information is quite important.

File: content/help/manual/create_account/index.md [hunk 2]
```diff
@@ -3,106 +3,77 @@ title: Creating An Account
...
-Don't like your suggested password? No problem! You can shuffle through more
-random phrases by clicking **Try Another One** until you find one you like
-(the generator has been known to spit out some hilarious combinations from time
-to time 😋). If this still doesn't yield the desired result, you can choose
-a more hands-on approach by selecting clicking **Choose My Own**. This will present
-you with the option to pick a password entirely by yourself or you can bring up
-a password generator that will give you much more control over how the password
-is generated. You can even choose a different language or increase the number
-of words used (4 words should be good enough for most personal uses but if you
-want to be safe, choosing 5, 6 or 7 words will make your password even more secure).
```
Comment (MaKleSoft) [inline content/help/manual/create_account/index.md:92]: Again, why get rid of all this?

File: content/help/manual/create_account/index.md [hunk 3]
```diff
@@ -3,106 +3,77 @@ title: Creating An Account
...
 
-Creating a Padloc account is simple and takes only a few moments. To get
-started, you can either open the Padloc web app by going to
-https://web.padloc.app or by downloading and installing the app. The first
-thing you'll see is a login screen. Since you don't have an account yet, you
-can't login at this point. Instead, click on **Sign Up Now**.
+## **Step 1: Download the [App](https://padloc.app/downloads/) or visit the [Login Page](https://web.padloc.app/signup)**
 
-## Step 1: Enter your name and email address
+## **Step 2: Enter Your Email Address**
```
Comment (MaKleSoft) [inline content/help/manual/create_account/index.md]: Please don't use `**` in headers. Headers should have a consistent look and if we want them to look more bold we can do that with CSS :)

**Review by MaKleSoft — COMMENTED** [review]
Good work on creating the video and updating the screenshots! But I'm a bit confused as to why you removed most of the informational text, which for the most part is still relevant? Also, why does the video stop in the middle of the signup process?

File: content/help/manual/create_account/index.md (hunk 1 above)
Comment (0101Diana) [reply content/help/manual/create_account/index.md]: Most of this text got moved to the "Read Me" text box at bottom - consolidated it there.

File: content/help/manual/create_account/index.md (hunk 3 above)
Comment (0101Diana) [reply content/help/manual/create_account/index.md]: Have been removed.

File: content/help/manual/create_account/index.md (hunk 1 above)
Comment (0101Diana) [reply content/help/manual/create_account/index.md]: Have included more info regarding the password generator now.

**Comment by 0101Diana** [comment]
In response to comments:
1.	Got rid of bolded headers feature.
2.	I did accidentally forget to include in both video and text the area regarding the more parameters’ password generator.  Is now included.
3.	The area regarding tips, encryption, and notice Padloc doesn’t record Master PWs, were including in the text box at the bottom in the first pull.  I have separated them out as you'll see.  
4.	I would like to do one more demo video so it’s clean and goes all the way until the new account opens up (and if you have an audio recording you’d like to include for soundtrack – please email me), but I need you to delete the demo@padloc.app member or it might be cool to create an email for this:  demov.v4@padloc.app.  Just a thought.  The video is pieced together because I cannot create a duplicate account with demo@padloc.app and had to use my diana@padloc.app email to go through the generating a master password part.

**Comment by 0101Diana** [comment]
In response to comments:
1.	Got rid of bolded headers feature.
2.	I did accidentally forget to include in both video and text the area regarding the more parameters’ password generator.  Is now included.
3.	The area regarding tips, encryption, and notice Padloc doesn’t record Master PWs, were including in the text box at the bottom in the first pull.  I have separated them out as you'll see.  
4.	I would like to do one more demo video so it’s clean and goes all the way until the new account opens up (and if you have an audio recording you’d like to include for soundtrack – please email me), but I need you to delete the demo@padloc.app member or it might be cool to create an email for this:  demov.v4@padloc.app.  Just a thought.  The video is pieced together because I cannot create a duplicate account with demo@padloc.app and had to use my diana@padloc.app email to go through the generating a master password part.

**Pull request closed without merging**